evm_loader/program/src/entrypoint.rs

```diff
@@ -130,10 +130,64 @@ fn process_instruction<'a>(
             debug_print!("create_account done");
             
             let mut data = program_info.data.borrow_mut();
-            let account_data = AccountData {ether, nonce, trx_count: 0u64, signer: *funding_info.key, code_size: 0u32};
+            let account_data = AccountData {ether, nonce, trx_count: 0u64, signer: *funding_info.key, code_account: Pubkey::new_from_array([0u8; 32]), code_size: 0u32};
             account_data.pack(&mut data)?;
             Ok(())
         },
+        EvmInstruction::CreateProgramAccount {lamports, space, ether, nonce} => {
```

Comment:
Merge with CreateAccount instruction. You can check that user pass contract_info using next_account_info result.
Remove `space` argument.